root/Makefile

```
@@ -163,6 +163,11 @@ grpcui:
 run:: pre-run build
 	@DEVBOX_LOGFMT="$(LOGFMT)" ./scripts/shell-wrapper.sh air-runner.sh
 
+## watch-build:         watch source files and build them for linux. To be used with devspace using `devenv apps run --bin .`
+.PHONY: watch-build
+watch-build:
+	@DEVBOX_LOGFMT="$(LOGFMT)" GOOS="linux" SKIP_TRIMPATH="true" SKIP_STARTING_APP="true" DLV_PORT=42097 ./scripts/shell-wrapper.sh gobin.sh github.com/cosmtrek/air@f7ec5b5ac29540748772dd4deb24a39f81ba9a47 -c $(CURDIR)/.air.toml
```

```suggestion
	@DEVBOX_LOGFMT="$(LOGFMT)" GOOS="linux" SKIP_TRIMPATH="true" SKIP_STARTING_APP="true" DLV_PORT=42097 ./scripts/shell-wrapper.sh gobin.sh github.com/cosmtrek/air@v1.44.0 -c $(CURDIR)/.air.toml
```

We can tag air now. They have versions 🎉

---

For some reason the gobin is unhappy with it

---

`./scripts/shell-wrapper.sh gobin.sh github.com/cosmtrek/air@v1.44.0` cannot execute binary file: Exec format error"

---

That's... depressing. Also, no idea how that actually breaks that way. That's a little worrying.